src/typhoon_automator/simulation.py

```diff
@@ -25,10 +31,17 @@ def __init__(
         
         self._schedule = EventSchedule()
 
+        self.stop_signal = False
+        self.start_time: datetime = None
+        self.stop_time: datetime = None
+
     def initialize(
             self,
             scenario: Any):
-        raise NotImplementedError()
+        if scenario is None:
+            raise ValueError("Scenario cannot be None")
+
+        self.scenario = scenario
```

Comment:
FYI:  this is alright for now, but we're going to have to re-visit it later.  There's a design change we didn't get to discuss which will affect this part of the code (briefly: Simulation no longer owns the scenario, Orchestrator owns the scenarios, Simulation just initializes them)